examples/secp256k1-transfer/package.json

```diff
@@ -5,21 +5,32 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "parcel src/index.html",
     "lint": "tsc --noEmit"
   },
   "keywords": [],
   "author": "",
   "license": "MIT",
   "dependencies": {
     "@ckb-lumos/lumos": "0.18.0",
+    "@headlessui/react": "^1.6.6",
+    "@heroicons/react": "^1.0.6",
+    "@mui/material": "^5.10.1",
     "@types/react": "^17.0.34",
     "@types/react-dom": "^17.0.11",
+    "bulma": "^0.9.4",
+    "nanoid": "^4.0.0",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "react-use": "^17.4.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.16.0",
     "@babel/core": "^7.16.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.16.0"
+    "@babel/plugin-transform-modules-commonjs": "^7.16.0",
+    "parcel": "^2.7.0",
+    "postcss": "^8.4.16",
+    "process": "^0.11.10",
+    "typescript": "^4.7.4"
```

Comment:
can we remove these deps? maybe we can use the `examples/package.json` to keep the example's env same with others

Reply:
We can not remove it. When I remove typescript dependency, these code can not run in codesandbox.

Reply:
even so, please use the same version of TypeScript with [package.json](https://github.com/nervosnetwork/lumos/blob/635ca26c8b9daa5809a0afbe38b5a2ca462dda18/package.json#L43).
I found examples work in [codesandbox](https://codesandbox.io/s/github/nervosnetwork/lumos/tree/develop/examples/secp256k1-transfer?file=/package.json) now

Reply:
Sorry, when I remove it, it can works well. The result of "Can not work" is from other reason.
https://codesandbox.io/s/github/IronLu233/lumos/tree/secp256k1-transfer/examples/secp256k1-transfer

Reply:
> ~~Sorry, when I remove it, it can works well. The result of "Can not work" is from other reason. https://codesandbox.io/s/github/IronLu233/lumos/tree/secp256k1-transfer/examples/secp256k1-transfer~~
These dependencies are needed when run it on local